docs/roadmap.md

```diff
@@ -96,35 +96,34 @@ For questions or status updates on either features listed here, or features not
-| Mica                                     |                                 |                                |                   | &#128994;                         |                       |
-| XAML Islands                                           |                                 |                                |                    | &#128994;                         |
-| Media Controls (e.g. `<MediaPlayerElement>`)             |                                 |                                |                              | &#128994;                         |
-| `<InkCanvas>`                                            |                                 |                                |                              | &#128994;                         |
-| `<MapControl>`                                           |                                 |                                |                              |  &#129002;                     |
+**WinUI 3 - Windows App SDK 1.0** is our latest release. 
+
+| Customer Capability                                    | Project Reunion 0.5 <br/>(March 2021) | Windows App SDK 0.8 <br/>(June 2021) | Windows App SDK 1.0 <br/> | Windows App SDK 1.1 (still under discussion) <br/> | Planned for a future update |
+| ------------------------------------------------------ | -------------------------------- | ------------------------------- | ----------------------------- | --------------------------- | --------------------------- |
+| Supported in any app using the Windows App SDK         | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Contains new Windows 11 controls/styles from WinUI 2.6 |            |            | &#128994;                       | &#128994;          | &#128994;     |
+| Supports MSIX Deployment                               | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Supports Unpackaged (non-MSIX) Deployment              |            |            | &#128994;                       | &#128994;          | &#128994;     |
+| Works downlevel to Windows 10 version 1809 and above   | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Supports the latest .NET                               | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| ARM64 support                                          | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| `<SwapChainPanel> `                                    | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Intellisense, Hot Reload, Live Visual Tree             | &#128311; | &#128311; | &#128994; (packaged apps only)  | &#128994;          | &#128994;     |
```

Comment:
For the "(packaged apps only)" comment, will that make the row width super wide?  Another way to do this would be to put an asterisk '*' beside it, and make the note below the table.  I'll let you decide which one you like best.